presto-native-execution/src/test/java/com/facebook/presto/nativeworker/AbstractTestNativeAggregations.java

```diff
@@ -201,6 +201,14 @@ public void testMinMaxBy()
 
         assertQuery("SELECT max_by(orderkey, extendedprice), max_by(orderkey, cast(extendedprice as REAL)) FROM lineitem");
         assertQuery("SELECT min_by(orderkey, extendedprice), min_by(orderkey, cast(extendedprice as REAL)) FROM lineitem where shipmode='MAIL'");
+
+        // 3 argument variant of max_by
+        assertQuery("SELECT max_by(orderkey, extendedprice, 10) FROM lineitem");
```

Comment:
Let's combine min_by and max_by in a single query to reduce test runtime. Let's also add group-by queries.

> // TODO(spershin): Need to add use cases with non-numeric types when implemented.

Let's resolve this TODO as well.